contracts/LifToken.sol

```diff
@@ -12,4 +12,54 @@ contract LifToken is MintableToken, Pausable {
 
   // Token decimals
   uint public constant DECIMALS = 18;
+
+  // Edit of the ERC20 token events to support data argument
```

Comment:
"Extra events based in ERC20 events"
and for functions:
"Extra methods transferData based on ERC20 transfer method, transferData allow the user to transfer value+data on transaction"
what do you think @nberger ?